.gitignore

```diff
@@ -113,10 +112,6 @@ ENV/
 # mypy
 .mypy_cache/
 
-# sqlite
-db.sqlite3
-db.sqlite3-journal
-
```

Comment:
Why are we removing these from gitignore?

Reply:
because of the move from their default location (under quipucords/quipucords) to ./var, which is already on .gitignore

Reply:
oh nice I didn't catch that!